src/components/sections/CommentFactory.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState, useEffect } from 'react';
+import Parse from 'parse/dist/parse.min.js';
+import '../../styles/index.css'
+import { List } from 'antd';
+import { Container } from 'react-bootstrap';
+
+
+function CommentFactory({ locationId }) {
+    const [readResults, setReadComments] = useState([]);
+
+    useEffect(() => {
+        const readComment = async function () {
```

Comment:
`readComments` - because it's multiple of them!

Reply:
Good point, we changed it!